docs/source/auth/index.rst

```diff
@@ -394,7 +394,11 @@ Permissions Database
 --------------------
 
 All users and permissions are stored in a database in ``basic_auth.db``, relative to the directory where MLflow server is launched.
-The location can be change in the :ref:`configuration <configuration>` file.
+The location can be change in the :ref:`configuration <configuration>` file. To run migrations, use the following command:
+
+.. code-block::
+
+    python -m mlflow.server.auth migrate --url <database_url>
```

Comment:
CLI like `mlflow auth ...` might make more sense.

Reply:
This is okay for now

Reply:
I'm actually not sure if we really need this because the database is automatically migarted.

Reply:
We might need this in case migration fails?

Reply:
> This is okay for now

Make sense, it should be ok as long as the migration works.